pyproject.toml

```diff
@@ -31,8 +31,12 @@ httpx = "^0.18.2"
 hypothesis = "^6.14.0"
 black = "^21.6b0"
 watchgod = "^0.7"
-
+taskipy = "^1.9"
 
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.taskipy.tasks]
+lint = "pre-commit run --all-files"
+revision = "docker-compose exec web alembic revision --autogenerate -m"
```

Comment:
This doesn't work if we are running the database locally (without docker).

Reply:
It works, if you add the database url of your local database to an '.env' file , hence the alembic environment uses the pydantic config.

Reply:
How would it work? It runs `docker-compose exec`, I am not runnning with docker...

Reply:
Oh, yes sorry. 
However, I think if you run your database locally ( which is way harder than just issuing a simple command, but let's go with the example) you can just simply do `poetry run alembic revision [...]` (and of course setting the env vars)  The PR and this task was meant to solve the problem of creating migrations using docker, that's why the migrations folder was mounted.

Reply:
We can just do it like how we do it on site, 
```toml
revision = "alembic revision --autogenerate -m"
docker-revision = "docker-compose exec web alembic revision --autogenerate -m"
```

Reply:
Yea, this is purely for a docker setup. I would expect most people developing this to be running in docker, due to the psql need. Rather than running their own psql server.

So I'd suggest keeping this as the default and then having local-revision as the alternative for people who want to run psql locally.